test-suite/js/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/DurationTest.scala

```diff
@@ -622,22 +622,28 @@ object DurationTest extends JasmineTest with ExpectExceptions {
 
     it("should respond to `addTo`") {
       val t = LocalTime.NOON
+      val d = LocalDate.MIN
```

Comment:
I'm afraid `LocalDate` and `LocalTime` are the only implemented temporals yet. The more obscure classes like `DayOfWeek` and `Month` are only subclasses of `TemporalAccessor`.